recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 230f80e064d905c49a1941af1b7e806e2f22b3c9a90ad5c21fd17d72636ea277
 
 build:
-  number: 0
+  number: 1
```

Comment:
You don't need this b/c you only want py3.7, right? There are no functional changes on the py<3.7 packages here.

Reply:
You are right, functionally there are no changes to the existing versions. This only adds 3.7.